docs/src/pages/guides/notifications.md

```diff
@@ -1,3 +1,214 @@
-# Notifications
+---
+id: notifications
+title: Notifications
+---
 
-under construction
+## Notification Types
+
+In `Monika`, we provide multiple notification channels. Notifications are available through sendgrid, mailgun, smtp and webhook.
+
+## Configurations
+
+Notifications can be sent through multiple channels and mail accounts, each are defined through configuration such as in the file `config.json.example` below. All you need for having a notification is set these configurations.
```

Comment:
You can remove this part: "All you need for having a notification is set these configurations."